reportportal/templates/jobs-deployment.yaml

```diff
@@ -19,6 +19,34 @@ spec:
     spec:
       containers:
       - env:
+      - name: RP_AMQP_HOST
+          value: "{{ .Values.rabbitmq.endpoint.address }}"
+        - name: RP_AMQP_PORT
+          value: "{{ .Values.rabbitmq.endpoint.port }}"
+        - name: RP_AMQP_USER
+          value: "{{ .Values.rabbitmq.endpoint.user }}"
+        - name: RP_AMQP_PASS
+        {{ if .Values.rabbitmq.endpoint.cloudservice }}
```

Comment:
There is no `cloudservice` parameter now. Please use this approach: https://github.com/reportportal/kubernetes/blob/master/reportportal/templates/api-deployment.yaml#L48